test/jdk/java/awt/event/MouseEvent/ClickDuringKeypress/ClickDuringKeypress.java

```diff
@@ -21,116 +21,91 @@
  * questions.
  */
 
+import java.awt.AWTException;
+import java.awt.EventQueue;
+import java.awt.Frame;
+import java.awt.Robot;
+import java.awt.event.InputEvent;
+import java.awt.event.KeyEvent;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.awt.event.WindowAdapter;
+import java.awt.event.WindowEvent;
+import java.awt.event.WindowEvent;
+
 /*
-  @test 1.2 98/08/05
+  @test
   @key headful
   @bug 4515763
   @summary Tests that clicking mouse and pressing keys generates correct amount of click-counts
-  @author andrei.dmitriev: area=awt.mouse
   @run main ClickDuringKeypress
 */
 
-/**
- * ClickDuringKeypress.java
- *
- * summary:
- */
-
-import java.applet.Applet;
-import java.awt.*;
-import java.awt.event.*;
-
-public class ClickDuringKeypress implements MouseListener
- {
+public class ClickDuringKeypress implements MouseListener {
    //Declare things used in the test, like buttons and labels here
```

Comment:
Comments can be removed.

Reply:
I thought about it already, seemed harmless, but I can remove it.